frontend/src/components/account/Auth.tsx

```diff
@@ -8,29 +8,24 @@ import Snackbar from '@material-ui/core/Snackbar';
 import TextField from '@material-ui/core/TextField';
 import { Location } from 'history';
 import React from 'react';
-import { useDispatch } from 'react-redux';
```

Comment:
(Related to the commit message)

Use the CamelCase components' names.
Capitalize OIDC.
Add a space between OIDC and `(`.